src/components/events-list/events-list.spec.jsx

```diff
@@ -121,7 +121,7 @@ describe('EventsList component', () => {
       const mockEvent = mockEvents.slice(1, 2)[0];
       const heading = EventsList.formatHeading(mockEvent);
 
-      expect(heading).toEqual(`11/24/17 7:00PM @ ${mockEvent.venue.city}`);
+      expect(heading).toContain(`@ ${mockEvent.venue.city}`);
```

Comment:
does the regex used in the other spec also apply here too?

Reply:
Added second eval to match regex as well.